test/e2e/app/app.go

```diff
@@ -161,7 +161,7 @@ func DefaultConfig(dir string) *Config {
 }
 
 // LaneDefinitions returns the (constant) list of lanes and their priorities.
-func LaneDefinitions(lanes map[string]uint32) (map[string]uint32, []uint32) {
+func LaneDefinitions(lanes map[string]uint32) (map[string]uint32, []payload.Lane) {
```

Comment:
This function could be split in two because its two results are used separately.

Reply:
I would leave them as is atm for the MVP and we can see down the line if we want to change this.